src/commands/up.rs

```diff
@@ -199,17 +186,21 @@ impl UpCommand {
             TriggerExecOpts::NoApp => {
                 cmd.arg("--help-args-only");
             }
-            TriggerExecOpts::App {
-                locked_app_url,
+            TriggerExecOpts::Local { app, working_dir } => {
+                let locked_app = spin_trigger::locked::build_locked_app(app, &working_dir)?;
+                let locked_url = self.write_locked_app(&locked_app, &working_dir).await?;
```

Comment:
I'm not keen on the lockfile builder being moved inside `run_trigger`.  But if that's the quick way out of this issue then we can do it as a fix and revisit the large-scale structure later.

Reply:
Yeah that's where it was before the merge. Just seems easiest until there is time for a proper refactor.